### ISSUE TITLE
Speed up folder upload/tarring process for front end CICD

### ISSUE THREAD

**Issue by vidjovanovic** [department-of-veterans-affairs/va.gov-team#77882]
Currently the CI.yaml has a stage where is upload folder contents to S3 in tar'd format, then downloads and runs them individually for each test in the follow step. This can be sped up by running the upload in parallel and using the existing folder in the previous step.

**Comment by vidjovanovic** [comment]
Researched best way to use folder from previous steps instead of upload/download artifact. Will run upload in parallel with separate step while transferring folder to next step.

**Comment by vidjovanovic** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/28641

Have PR up for CI improvement

**Comment by vidjovanovic** [comment]
Spoke with Josh regarding caching. Cache actions did not work for transferring folders between build jobs.

**Comment by vidjovanovic** [comment]
Enabled caching in every step except for cypress, will have to take out element by element until I see what is the offending line.

**Comment by jhouston0** [comment]
closing issue as platform group refactored the ci/cd pipeline and this is no longer needed
